CSS/styles.css

```diff
@@ -0,0 +1,434 @@
+}
+
+.btn:hover {
+    box-shadow: rgba(0, 0, 0, 0.07) 0px 4px 6px;
+}
+
+.btn-rounded {
+    border-radius: 4px;
+}
+
+/* ================================ Header ================================ */
+/* ---------------------------- Header Top Left --------------------------- */
+header {
+    width: 100%;
+    height: 178px;
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 100;
+    background-color: #FFF;
+}
+
+.header-top {
+    border-bottom: 1px solid #DDD;
+}
+
+.header-top-container {
+    height: 80px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.header-logo img {
+    width: 107px;
+}
+
+/* ---------------------------- Header Top Middle --------------------------- */
+.header-button-container {
+    display: flex;
+    height: 48px;
+    text-align: center;
+    border-radius: 30px;
+    align-items: center;
+}
+
+.header-button {
+    padding: 14px 18px 19px;
+    font-size: 15px;
+}
+
+.button-separator-line {
+    flex: 0 0 1px;
+    height: 24px;
+    width: 1px;
+    background-color: #DDD;
+}
+
+/* ---------------------------- Header Top Right --------------------------- */
+.header-right-container {
+    display: flex;
+    align-items: center;
+}
+
+.become-host-link {
+/*    width: 123px;*/
+    height: 42px;
+    border-radius: 30px;
+    border: none;
+    padding: 14px 15px 16px;
+    font-size: 14px;
+}
+
+.become-host-link:hover {
+    background-color: #F7F7F7;
+}
+
+.choose-language {
+    width: 40px;
+    height: 42px;
+    padding: 13px;
+    margin-left: -7px;
+}
+
+.choose-language img {
+    width: 16px;
+    height: 16px;
+}
+
+.choose-language:hover {
+    background-color: #F7F7F7;
+    border-radius: 22px;
+}
+
+.signup {
+    width: 77px;
+    height: 42px;
+    border: 1px solid #DDD;
+    border-radius: 30px;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+}
+
+img#menu {
+    width: 12px;
+    height: 12px;
+}
+
+img#account {
+    width: 30px;
+    height: 30px;
+}
+
+/* --------------------------- Header Bottom Half --------------------------- */
+.header__bottom {
+    z-index: 101;
+    width: 90%;
+    margin: auto;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 20px;
+    flex-wrap: nowrap;
+}
+
+.carousel-wrapper {
+    width: 90%;
+    height: 78px;
+    position: relative;
+    overflow: hidden;
+    display: flex;
+    align-items: center;
+}
+
+.carousel-arrow {
+    width: 28px;
+    height: 28px;
+    border-radius: 14px;
+    text-align: center;
+    line-height: 26px;
+}
+
+.carousel-category-container {
+    width: 100%; /* this fixed the issue with the arrows getting squished */
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: nowrap;
+    column-gap: 34px;
+    overflow: scroll;
+    scroll-behavior: smooth;
+    scrollbar-width: none; /* Firefox */
+    -ms-overflow-style: none;  /* Internet Explorer 10+ */
```

Comment:
IE support is rare today. Probably not something you will have to think about. Also, there are tools now that will help you with the vendor prefix and compatibility stuff like https://www.npmjs.com/package/autoprefixer